2022-01-14 13:42:57,649 INFO    MainThread:416335 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 13:42:57,649 INFO    MainThread:416335 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:42:57,650 INFO    MainThread:416335 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:42:57,650 INFO    MainThread:416335 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_134257-2dnkvagv/logs/debug.log
2022-01-14 13:42:57,650 INFO    MainThread:416335 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_134257-2dnkvagv/logs/debug-internal.log
2022-01-14 13:42:57,650 INFO    MainThread:416335 [wandb_init.py:init():404] calling init triggers
2022-01-14 13:42:57,650 INFO    MainThread:416335 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 13:42:57,650 INFO    MainThread:416335 [wandb_init.py:init():460] starting backend
2022-01-14 13:42:57,650 INFO    MainThread:416335 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 13:42:57,655 INFO    MainThread:416335 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 13:42:57,658 INFO    MainThread:416335 [backend.py:ensure_launched():221] started backend process with pid: 416394
2022-01-14 13:42:57,659 INFO    MainThread:416335 [wandb_init.py:init():469] backend started and connected
2022-01-14 13:42:57,662 INFO    MainThread:416335 [wandb_init.py:init():533] updated telemetry
2022-01-14 13:42:57,670 INFO    MainThread:416335 [wandb_init.py:init():563] communicating current version
2022-01-14 13:42:58,031 INFO    MainThread:416335 [wandb_init.py:init():568] got version response 
2022-01-14 13:42:58,031 INFO    MainThread:416335 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 13:42:58,218 INFO    MainThread:416335 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 13:42:58,273 INFO    MainThread:416335 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 13:42:58,273 INFO    MainThread:416335 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 13:42:58,274 INFO    MainThread:416335 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 13:42:58,275 INFO    MainThread:416335 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 13:42:58,275 INFO    MainThread:416335 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 13:42:58,966 INFO    MainThread:416335 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 13:42:58,966 INFO    MainThread:416335 [wandb_run.py:_restore():1752] restore
2022-01-14 13:43:01,299 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:43:01,407 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:43:01,557 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:43:02,226 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 2141
}

2022-01-14 13:43:02,328 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 9498
}

2022-01-14 13:43:02,429 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:02,530 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:02,631 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:02,733 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:02,834 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:02,937 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:03,292 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}

2022-01-14 13:43:03,759 INFO    MainThread:416335 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9498
  total_bytes: 9498
}
local_info {
}

2022-01-14 13:43:04,811 INFO    MainThread:416335 [wandb_run.py:_append_files():2180] logging synced files
